### PR TITLE
Add contributing information to packages

### PR DESCRIPTION
**Pull request by talldan**
## Description
One of the valid ways to find out about an individual Gutenberg package is via npm (https://www.npmjs.com/package/@wordpress/components). From there users can visit a package homepage (which most prominently shows the package's README file) to find out more about a package (https://github.com/WordPress/gutenberg/blob/trunk/packages/components/README.md). Some visitors may wish to make a contribution, but our package READMEs don't contain links to the contributor guide.

It'd be nice to rectify that to improve the contribution experience.

Here I've started off by adding details to one package (a11y, because its first in the alphabetical list). I'm looking for feedback on the text. Once that's honed, I'll copy and paste it across to the other packages.

## Types of changes
Documentation

### CODE REVIEW

**Comment by kevin940726** [comment]
Thank you for starting this!

Maybe we should also include this in the general [guideline](https://github.com/WordPress/gutenberg/blob/HEAD/packages/README.md#creating-a-new-package) when creating a new package as well.

**Comment by gziolo** [comment]
Great idea. I guess we need only to agree on the wording and apply it to all existing packages.

 ❤️

File: packages/a11y/README.md [hunk 1]
```diff
@@ -70,4 +70,10 @@ For context I’ll quote [this article on WordPress.org](https://make.wordpress.
 
 See <https://make.wordpress.org/design/handbook/design-guide/browser-support/>
 
+## Contributing to this package
+
+This package is part of the Gutenberg project. The repository is organized as a monorepo and contains many packages.
```
Comment (gziolo) [inline packages/a11y/README.md]: We could also make it clear that this monorepo contains all WordPress packages used in the core. Maybe also link to https://make.wordpress.org/core/ in some way to make that connection explicit.

**Comment by anton-vlasenko** [comment]
LGTM. That's a good improvement!

**Review by getdave — COMMENTED** [review]
Looks like a good change. Let me know if I can help with rolling this out to the other packages as I can imagine that will be a lot of copy/paste! 😄

**Comment by talldan** [comment]
I've changed the wording a bit. Let me know what you think.

**Review by getdave — COMMENTED** [review]
Still reads well to me 👍

**Comment by gziolo** [comment]
It looks great ❤️

**Review by gziolo — APPROVED** [review]

File: packages/README.md [hunk 2]
```diff
@@ -46,6 +46,7 @@ When creating a new package, you need to provide at least the following:
     - Installation details
     - Usage example
     - API documentation, if applicable ([more info](#maintaining-api-documentation))
+    - A link to the contributing guidelines
```
Comment (gziolo) [inline packages/README.md]: Should we put the full text of the new section proposed so people can copy and paste it?

Comment (kevin940726) [reply packages/README.md]: Maybe something like a script to auto-generate a new package would be better? Not something should be covered in this PR though but just an idea 🙈 .

**Review by kevin940726 — APPROVED** [review]
LGTM 👍

**Review by ryanwelcher — APPROVED** [review]
LGTM 👍

**Review by juanmaguitar — APPROVED** [review]
I completely agree with these type of improvements. I think there's a lot still to do in terms of adding context to the different lines of resources generated from the WordPress Core ecosystem. I find this specific change SUPER useful and I'd love to see more like this one.
Thank you @talldan for taking care of this.

File: packages/README.md (hunk 2 above)
Comment (talldan) [reply packages/README.md]: I've added a link to the text in one of the packages. 👍

**Comment by talldan** [comment]
Thanks for the reviews everyone. I've added copy/pasted the text across to all packages now and will merge when CI passes 👍

**Pull request merged**